pkg/localize/locales/en/cmd/kafka_delete.en.toml

```diff
@@ -51,4 +47,4 @@ one = 'Deleting Kafka instance'
 
 [kafka.delete.log.info.deleteSuccess]
 description = 'Info message when instance was deleted'
-one = 'Kafka instance "{{.Name}}" has successfully been deleted'
\ No newline at end of file
+one = 'Kafka instance "{{.Name}}" is being deleted. You can monitor its progress by running "rhoas status".'
```

Comment:
When you delete a Kafka instance, it will remove it from the config file, so `rhoas status` will not show anything. Also, if you delete a Kafka by passing the ID or name, `rhoas status` will be showing the status of a different one.

Reply:
Best to remove the second sentence.